### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by mrombout** [mrombout/hugo-protagonist#2]
In order to ensure every visitor will be able to access the content on websites using this theme, special care should be taken to make the theme follow proper accessibility practices.

https://developer.mozilla.org/en-US/docs/Web/Accessibility
